unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs

```diff
@@ -102,6 +124,21 @@ public void ReportScene()
         WebInterface.SendReportScene($"{coords.x},{coords.y}");
     }
 
+    public void SetHomeScene(bool isOn)
+    {
+        var coords = playerCoords.Get();
+        if (playerCoords == homeCoords)
+        {
+            if (!isOn)
+                WebInterface.SetHomeScene($"0,0");
```

Comment:
please instead of referencing `WebInterface` directly, we should inject an interface that sets the home scene, so follow the dependency inversion principle

Reply:
I don't get this, didn't we always use the WebInterface static class in this way like I did?

Reply:
you can create a new interface then inject it in `MinimapHUDController`, for example:
```
interface IHomeLocationController
{
    void SetHomeScene(Vector2 location);
}
```

you can have a single implementation:
```
class WebInterfaceHomeLocationController : IHomeLocationController
{
    public void SetHomeScene(Vector2 location)
    {
        WebInterface.SetHomeScene($"{location.x},{location.y}");
    }
}
```

this way we remove the coupling with `WebInterface`, its easier to mock/test and opened to change the way it works in the future without modifying `MinimapHUDController`